lib/faraday/adapter/typhoeus.rb

```diff
@@ -7,6 +7,11 @@ def call; end
 
       dependency 'typhoeus'
       dependency 'typhoeus/adapters/faraday'
+
+      def initialize(adapter_options = {})
```

Comment:
You're right, this class was still around for typhoeus <= 1.3.0, however I changed adapters constructor to not accept the `@app` anymore (as they don't need to call `@app.call(env)`) and this breaks even the latest version of the adapter in the gem. Here I'm overriding the `initialize` method so that it works as expected. The only other solution is to revert the adapter initialzer signature and pass `@app` anyway although useless.